frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -213,88 +216,102 @@ export const PersonalDashboard = () => {
                 most of Unleash
             </ScreenExplanation>
             <StyledHeaderTitle>Your resources</StyledHeaderTitle>
-            <ContentGrid container columns={{ lg: 12, md: 1 }}>
-                <SpacedGridItem item lg={4} md={1}>
-                    <Typography variant='h3'>My projects</Typography>
-                </SpacedGridItem>
-                <SpacedGridItem
-                    item
-                    lg={8}
-                    md={1}
-                    sx={{ display: 'flex', justifyContent: 'flex-end' }}
-                >
-                    <Badge color='warning'>Setup incomplete</Badge>
-                </SpacedGridItem>
-                <SpacedGridItem item lg={4} md={1}>
-                    <List
-                        disablePadding={true}
-                        sx={{ maxHeight: '400px', overflow: 'auto' }}
+            {noProjects ? (
+                <ContentGridNoProjects
+                    owners={[{ ownerType: 'system' }]}
+                    admins={[
+                        { name: 'admin' },
+                        {
+                            name: 'Christopher Tompkins',
+                            imageUrl:
+                                'https://avatars.githubusercontent.com/u/1010371?v=4',
+                        },
```

Comment:
This entire bit is largely just wrapping the old content in a ternary. we'll probably sort this out with the union type down the road, but this makes it work right now.